Crashlytics/Crashlytics/Public/FirebaseCrashlytics/FIRCrashlytics.h

```diff
@@ -191,9 +191,9 @@ NS_SWIFT_NAME(Crashlytics)
  * sendUnsentReports or deleteUnsentReports, depending on whether or not the user gives consent.
  *
  * Disable automatic collection by:
- *  - Adding the FirebaseCrashlyticsCollectionEnabled: NO key to your App's Info.plist
- *  - Calling [[FIRCrashlytics crashlytics] setCrashlyticsCollectionEnabled:NO] in your app
- *  - Setting FIRApp's isDataCollectionDefaultEnabled to NO
+ *  - Adding the FirebaseCrashlyticsCollectionEnabled: false key to your App's Info.plist
```

Comment:
This should still be NO, since plist values follow ObjC semantics.

Reply:
Makes sense, fixed